### PR TITLE
Add Version History for jQuery

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Comment by artem-kurchenko** [comment]
The info is different now, according to [package.json](https://github.com/DevExpress/DevExtreme/blob/20_2/package.json#L113) 20.1, 19.2, 19.1 support 3.5 or later

18.1, 18.2 have 3.4.1
https://github.com/DevExpress/DevExtreme/blob/18_1/package.json#L113

I think a more detailed table is necessary
@San4es  could you confirm my thoughts?

**Comment by nightskylark** [comment]
> The info is different now, according to [package.json](https://github.com/DevExpress/DevExtreme/blob/20_2/package.json#L113) 20.1, 19.2, 19.1 support 3.5 or later
> 
> 18.1, 18.2 have 3.4.1
> https://github.com/DevExpress/DevExtreme/blob/18_1/package.json#L113
> 
> I think a more detailed table is necessary
> @San4es could you confirm my thoughts?

There was a vulnerability in jquery@3.4.0, but DevExtreme should work with this version too.

**Review by nightskylark — APPROVED** [review]

**Review by artem-kurchenko — APPROVED** [review]
Okay, then I'm good :)

**Comment by San4es** [comment]
@artem-kurchenko
- The _recommended_ jQuery version is v3.5.1 due to security issues.
- Yes, we still _support_ jQuery v2.x and test against it.

**Review by San4es — COMMENTED** [review]
Let's mention the recommended jQuery version.

**Review by San4es — APPROVED** [review]
LGTM

File: concepts/58 jQuery Components/80 Version History.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            v18.2.x <br/>
+            v18.1.x <br/>
+            v17.2.x <br/>
+            v17.1.x
+        </td>
+        <td>v2.1 - v2.2 and v3.x</td>
+    </tr>
+</table>
+
+[note] Although earlier versions are supported, we recommend that you use jQuery v3.5.1 to guard against <a href="https://snyk.io/vuln/npm:jquery" target="_blank">vulnerabilities</a> found in those earlier versions.
```
Comment (albertov05) [inline concepts/58 jQuery Components/80 Version History.md]:
```suggestion
[note] Although earlier versions are supported, we recommend that you use jQuery v3.5.1 to protect against <a href="https://snyk.io/vuln/npm:jquery" target="_blank">vulnerabilities</a> found in earlier versions.
```

**Pull request merged**